packages/fxa-settings/src/index.tsx

```diff
@@ -64,11 +62,7 @@ try {
             bundles={['settings', 'react']}
             userLocales={navigator.languages}
           >
-            {showNewReactApp ? (
-              <App {...{ flowQueryParams }} />
-            ) : (
-              <Settings {...{ flowQueryParams }} />
```

Comment:
Q: Do we still need to import the `Settings` class (L12)?

Reply:
Nice, we don't. 😁

Man, we really need those auto plugin sort tickets done.